Resources/Prototypes/Entities/Structures/Power/Generation/Singularity/singularity.yml

```diff
@@ -50,3 +53,33 @@
   - type: WarpPoint
     follow: true
     location: singularity
+  - type: Sprite
```

Comment:
sprite is already defined a bit further up, you need to merge them

Reply:
fixed